### PR TITLE
bump deps

### PR DESCRIPTION
**Pull request by zroubalik**
Signed-off-by: Zbynek Roubalik <zroubali@redhat.com>

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Relates https://github.com/kedacore/keda/issues/2503

### CODE REVIEW

**Comment by zroubalik** [comment]
There will be a follow up PR once https://github.com/kubernetes-sigs/custom-metrics-apiserver/pull/97 is merged.

**Review by tomkerkhove — APPROVED** [review]

File: pkg/scalers/gcp_pub_sub_scaler.go [hunk 1]
```diff
@@ -244,7 +244,7 @@ func (s *pubsubScaler) getMetrics(ctx context.Context, metricType string) (int64
 func getSubscriptionData(s *pubsubScaler) (string, string) {
 	var subscriptionID string
 	var projectID string
-	regexpExpression, _ := regexp.Compile(compositeSubscriptionIDPrefix)
+	regexpExpression := regexp.MustCompile(compositeSubscriptionIDPrefix)
```
Comment (JorTurFer) [inline pkg/scalers/gcp_pub_sub_scaler.go:247]: Should we move this from here to a package scoped variable?

**Review by JorTurFer — APPROVED** [review]
LGTM, 
Only one small nit

File: pkg/scalers/gcp_pub_sub_scaler.go (hunk 1 above)
Comment (zroubalik) [reply pkg/scalers/gcp_pub_sub_scaler.go]: Good point, fixed.

**Comment by tomkerkhove** [comment]
Thanks! 🥇

**Comment by zroubalik** [comment]
> Thanks! 🥇

No need to thank. Most likely there will be a round two 🤦‍♂️😂

**Comment by JorTurFer** [comment]
❤️ ❤️ 🥇 ❤️ ❤️

**Pull request merged**